README.md

```diff
@@ -295,6 +330,27 @@ For more information and to see an example of this in use take a look at the [Sp
 
 Instructions on how to run the Spring example project can be found in the [yoti-sdk-spring-boot-example](/yoti-sdk-spring-boot-example) folder.
 
+## Breaking changes and enhancements made in v2.0.0
+
+As well as exposing significant new functionality, we've taken this opportunity to clean up and simply our public API.
+
+### Dropped support for Java 6
+Minimum supported Java version is now 7.
+
+### New interfaces - Attribute<T> and Anchor
+The old `com.yoti.api.client.Attribute` class has been replaced with `com.yoti.api.client.Attribute<T>`, which now exposes a `List<Anchor>` for the sources and verifiers of the attribute.
+The old `Set<String> getSources()` and `Set<String> getVerifiers()` methods have been replaced with `List<Anchor> getSources()` and `List<Anchor> getVerifiers()`.
+
+### Changes to HumanProfile and ApplicationProfile
```

Comment:
You can remove ApplicationProfile

Reply:
Well, ApplicationProfile did change too....

Reply:
:+1: